src/MLJMultivariateStatsInterface.jl

```diff
@@ -150,4 +150,1078 @@ metadata_pkg.(
+    covariance (used in computing between-class scatter matrix, Sb).
+- `out_dim::Int=0`: The output dimension, i.e dimension of the transformed space,
+    automatically set if 0 is given (default).
+- `regcoef::Float64=1e-6`: The regularization coefficient (default value 1e-6). A positive
+    value `regcoef * eigmax(Sw)` where `Sw` is the within-class scatter matrix, is added
+    to the diagonal of Sw to improve numerical stability. This can be useful if using
+    the standard covariance estimator.
+- `dist::SemiMetric=SqEuclidean`: The distance metric to use when performing classification
+    (to compare the distance between a new point and centroids in the transformed space),
+    an alternative choice can be the `CosineDist`.Defaults to `SqEuclidean`.
+
+# Operations
+
+- `transform(mach, Xnew)`: Return a lower dimentional projection of the input `Xnew` having the same scitype as `X` above.
+- `predict(mach, Xnew)`: Return predictions of the target given
+  features `Xnew` having the same scitype as `X` above. Predictions
+  are probabilistic.
+- `predict_mode(mach, Xnew)`: Return the modes of the probabilistic predictions
+   returned above.
+
+
+# Fitted parameters
+
+The fields of `fitted_params(mach)` are:
+
+- `projected_class_means`: The matrix comprised of class-specific means as
+  columns (of size `(d,m)`), where d corresponds to input features and m corresponds to class.
+- `projection_matrix`: The matrix used to project `X` into a lower dimensional space.
+
+# Report
+
+The fields of `report(mach)` are:
+
+- `classes`: The classes seen during model fitting.
+- `out_dim`: The dimensions the model is projected to.
+- `class_means`: The matrix comprised of class-specific means as
+  columns (of size `(d,m)`), where d corresponds to input features and m corresponds to class.
+- `mean`: The mean vector.
+- `class_weights`: The weights of each class.
+- `Sb`: The between class scatter matrix.
+- `Sw`: The within class scatter matrix.
+- `nc`: The number of classes.
+
+# Examples
+
+```
+using MLJ
+
+LDA = @load LDA pkg=MultivariateStats
+
+X, y = @load_iris
+
+model = LDA()
+mach = machine(model, X, y) |> fit!
+
+Xproj = transform(mach, X)
+y_hat = predict(mach, x)
+labels = predict_mode(mach, x)
+```
+
+See also
+[`BayesianLDA`](@ref), [`SubspaceLDA`](@ref), [`BayesianSubspaceLDA`](@ref)
+"""
+LDA
+"""
+$(MMI.doc_header(BayesianLDA))
+
+`BayesianLDA`: Bayesian Multiclass linear discriminant analysis. The algorithm
+learns a projection matrix `P` that projects a feature matrix `Xtrain` onto a lower
+dimensional space of dimension `out_dim` such that the trace of the transformed
+between-class scatter matrix(`Pᵀ*Sb*P`) is maximized relative to the trace of the
+transformed within-class scatter matrix (`Pᵀ*Sw*P`). The projection matrix is scaled
+such that `Pᵀ*Sw*P = n` or `Pᵀ*Σw*P=I` (Where `n` is the number of training samples
+and `Σw` is the within-class covariance matrix).
```

Comment:
The first part here can be simplified exactly as for vanilla LDA as noted above (there's no difference in how the transformations are learned).